src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -123,6 +127,12 @@ private function executeCallback(callable $callback, ContainerConfigurator $cont
                 case self::class:
                     $arguments[] = $this;
                     break;
+                case 'string':
+                    if ('env' === $parameter->getName()) {
```

Comment:
$this->env can be null so we might also what it's not here

Reply:
Done